include/ignition/physics/GetEntities.hh

```diff
@@ -213,6 +213,56 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature retrieves the nested model pointer from the parent
+    /// model by specifying the model index and model index/name.
+    class IGNITION_PHYSICS_VISIBLE GetNestedModelFromModel
+        : public virtual FeatureWithRequirements<GetModelFromWorld>
+    {
+      public: template <typename PolicyT, typename FeaturesT>
+      class Model : public virtual Feature::Model<PolicyT, FeaturesT>
+      {
+        // typedefs for the type of Model that this Model can return
+        public: using ModelPtrType = ModelPtr<PolicyT, FeaturesT>;
+        public: using ConstModelPtrType = ConstModelPtr<PolicyT, FeaturesT>;
+
+        /// \brief Get the number of Models inside this Model.
+        public: std::size_t GetModelCount() const;
+
+        /// \brief Get a Model that exists within this Model.
+        /// \param[in] _index
+        ///   Index of the model within this model.
+        /// \return A model reference. If _index is GetModelCount() or higher,
+        /// this will be a nullptr.
+        public: ModelPtrType GetModel(std::size_t _index);
```

Comment:
I would suggest making this function more explicit by calling it `GetNestedModel`. Same for the overloads of `GetModel` below.

Reply:
Done in https://github.com/ignitionrobotics/ign-physics/pull/228/commits/d137bba029a5cea24d22ee5458ef32b740e32fcf. Also made the same change to `ConstructEmptyNestedModel`.